job-dsls/jobs/FlowJob.groovy

```diff
@@ -451,9 +451,9 @@ matrixJob("jbpmTestContainerMatrix-kieAllBuild-${kieMainBranch}") {
     }
 
     publishers {
-        wsCleanup()
-        archiveJunit("**/TEST-*.xml")
+        archiveJunit("**/target/failsafe-reports/TEST-*.xml")
```

Comment:
I think this notation is the same, not sure why it should not be ;)

Reply:
it is - it is - but I like to have it all the same notation :)